studio/components/to-be-cleaned/Storage/StorageExplorer/FileExplorerHeader.js

```diff
@@ -313,13 +313,13 @@ const FileExplorerHeader = ({
               <Dropdown.RadioGroup key="sortOptions" value={sortBy} onChange={onChangeSortBy}>
                 <Dropdown.Radio value={STORAGE_SORT_BY.NAME}>Sort by name</Dropdown.Radio>
                 <Dropdown.Radio value={STORAGE_SORT_BY.CREATED_AT}>
-                  Sort by last created
+                  Sort by create time
```

Comment:
Hung up on the "Sort by create time" → maybe should be "Sort by _created_ time"? Since the one after is past-tense (modified).  Could also move the "Sort by" to the button and simplify the dropdown.  
<img width="349" alt="CleanShot 2022-08-19 at 09 37 13@2x" src="https://user-images.githubusercontent.com/105593/185615342-a0df3f5f-8f17-47cc-949c-582411853b31.png">
.